### PR TITLE
Add Docker Compose to help new users

### PR DESCRIPTION
**Pull request by enric1994**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+version: '2.3'
+
+services:
+
+  gym_mupen64plus:
```
Comment (kevinhughes27) [inline docker-compose.yml]: docker-compose will use this name as a prefix already by default, how about a more descriptive name like `agent` then the full container name would be `gym_mupen64plus_agent`

Comment (kevinhughes27) [reply docker-compose.yml]: same with  `container_name` below

File: example.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import gym, gym_mupen64plus
+import subprocess
+
+def startVNC(display):
+    subprocess.call("x11vnc -display :{} -localhost -forever -viewonly &".format(str(display)), shell=True)
+
+env = gym.make('Mario-Kart-Luigi-Raceway-v0')
+env.reset()
+
+#Start casting VNC at display 1
```
Comment (kevinhughes27) [inline example.py]: nit
```suggestion
# Start casting VNC at display 1
```

**Comment by bzier** [comment]
Hey @enric1994, sorry I haven't reviewed this yet, or provided feedback. I'll try to get to it soon, possibly this weekend.

**Comment by bzier** [comment]
Closing in favor of #74 
Thank you for this contribution, but I chose to take a slightly different approach on this. I have wanted to split out the separate processes for a while, and this seemed like a good opportunity. Feel free to try out and provide feedback on that PR.

**Pull request closed without merging**